File: Java/Busqueda_secuencial.java
public class Busqueda_secuencial {

    static int findEle(int[] arreglo, int largo, int elemento){
        for(int indice =0; indice < largo; indice++){
            if(arreglo[indice] == elemento){
                return indice;
            }
        }
        return -1;
    }
    public static void main(String[] args) {
        //Declaracion de arreglo
        int[] arreglo = {12, 34, 10, 6, 40, 89, 98, 57, 19, 69};
        int elemento = 40;
        int largo = arreglo.length; //variable que guarda el largo del arreglo

        //Envío el arreglo(contenido), el largo(cuantos elementos) y el elemento a buscar
        int indice = findEle(arreglo, largo, elemento);
    }
}

File: C++/Busqueda_secuencial.cpp
#include <iostream>

using namespace std;

//funcion de busqueda
int findEle(int arreglo[], int s, int elemento) {
    for (int indice = 0; indice < s; indice++) {
        if (arreglo[indice] == elemento) {
            return indice;
        }
    }
    return -1;
}

int main() {
    int arreglo[] = {12, 34, 10, 6, 40, 89, 98, 57, 19, 69};
    int elemento = 19;
    // Calculamos el tamaño del arreglo
    int s = sizeof(arreglo) / sizeof(arreglo[0]);
    
    //Mando el arreglo, el largo (s) y el elemento por buscar a la funcion  
    int indice = findEle(arreglo, s, elemento);
    
    if (indice != -1) {
        cout << "El elemento se encuentra en la posicion: " << (indice + 1) << endl;
    } else {
        cout << "No se encuentra el elemento" << endl;
    }
    
    return 0;
}